src/Wallet.purs

```diff
@@ -195,3 +291,28 @@ foreign import _attachSignature
   :: ByteArray -- CBOR bytes of tx
   -> ByteArray -- CBOR bytes of witness set
   -> Effect (ByteArray)
+
+-------------------------------------------------------------------------------
+-- Gero FFI stuff
+-------------------------------------------------------------------------------
+foreign import data GeroConnection :: Type
+
+foreign import _enableGero :: Effect (Promise GeroConnection)
+
+foreign import _getGeroAddress :: GeroConnection -> Effect (Promise String)
+
+foreign import _getGeroCollateral
+  :: MaybeFfiHelper -> GeroConnection -> Effect (Promise (Maybe String))
+
+getGeroCollateral :: GeroConnection -> Effect (Promise (Maybe String))
+getGeroCollateral = _getGeroCollateral maybeFfiHelper
+
+foreign import _signTxGero
+  :: String -- Hex-encoded cbor of tx
+  -> GeroConnection
+  -> Effect (Promise String)
+
+foreign import _submitTxGero
```

Comment:
We can remove this (see my comment above). In fact we should do the same for Nami as well